cartaya/social-page/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./resources/css/styles.css">
+    <title>Social Page</title>
+</head>
+<body class="view-port">
+    <header class="socialpg_header">
+        <img src="./resources/icons/s-web.svg" alt=" web icon">
+    </header>
+    <main class="socialpg_container lato-font">
+        <p class="socialpg_text-top">We are</p>
+        <h2 class="socialpg_title">A Social web</h2>
+        <p class="socialpg_text-about-us">We conect you with your friends in any site in all of the word. We work for you every time</p>
+        <div class="socialpg_circle-container">
+            <div class="socialpg_circle">
+                <p class="socialpg_circle-numbers">365</p>
+                <p class="socialpg_circle-time">Days</p>
+            </div>
+            <div class="socialpg_circle">
+                <p class="socialpg_circle-numbers">24</p>
+                <p class="socialpg_circle-time">Hours</p>
+            </div>
+            <div class="socialpg_circle">
+                <p class="socialpg_circle-numbers">60</p>
+                <p class="socialpg_circle-time">Minutes</p>
+            </div>
+            <div class="socialpg_circle">
+                <p class="socialpg_circle-numbers">60</p>
+                <p class="socialpg_circle-time">Seconds</p>
+            </div>
+        </div>
+        <form class="socialpg_form" action="#" method="post">
+            <label class="socialpg_text-befor-mail">
```

Comment:
this label tag is the label and should be the one connected to the one using the _for_ attribute